src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar__month-length--29">
+      <div class="calendar__day calendar__day--1 calendar__day-start--sun"></div>
```

Comment:
```suggestion
    <div class="calendar calendar__month-length--29 calendar__day-start--sun">
      <div class="calendar__day calendar__day--1 "></div>
```
It has to be together in block